modules/bootstrapped/src/generated/com/amazonaws/dynamodb/DynamoDB.scala

```diff
@@ -165,7 +165,9 @@ object DynamoDBOperation {
     val hints: Hints = Hints.empty
 
     final case class InternalServerErrorCase(internalServerError: InternalServerError) extends ListTablesError
+    def internalServerErrorCase(internalServerErrorCase:InternalServerError): ListTablesError = InternalServerErrorCase(internalServerErrorCase)
```

Comment:
thought: this is probably not as useful as it could be. According to [Oli's message from April](https://discord.com/channels/1045676621761347615/1045676622491164724/1093807442644701236), this method should take the same arguments as `InternalServerError` and wrap them into that case class itself. Otherwise, it's just a different casing of `InternalServerErrorCase`

Reply:
@kubukoz the benefit of this, as it is, is two-folds :

1. The return type is the union, whereas calling the ADT member constructor follows the type of the member, which is annoying in many aspects 
2. The names don't suck as much.

I'd be willing to add overloads in the case where the member targets a structure, but I'm satisfied with this current form

Reply:
I was referring to this part:

> with flatten constructor parameters when the union member targets a structure

so what you said you'd be willing to do ;) I thought that was a given